### ISSUE TITLE
SY6970 library does not change the actual charging rate.

### ISSUE THREAD

**Issue by Grex44** [Xinyuan-LilyGO/T-Display-S3-Long#24]
Hi, 

I'm experiencing some issue with the charging speed rate on the SY6970. I included the library from lilygo's repository. Even after setting up  _PMU.setChargerConstantCurr();_ to _PMU.setChargerConstantCurr(3200);_ The tested charging rate was still at 400mA. 

I've even increase the input current limit with _setInputCurrentLimit()_ to _setInputCurrentLimit(2000)_; 

There was no change in the actual charging performance.
 Appreciate your assistance in it.

**Comment by rhcp4life** [comment]
is the charging rate fixed based on this resistor?
![image](https://github.com/Xinyuan-LilyGO/T-Display-S3-Long/assets/6364405/3bbbbe00-a16c-4549-a8af-aae24147f8ed)

**Comment by Grex44** [comment]
Ok thanks. Is there anyone possible to point out which resistor is R24 on the board itself since there isn't any silkscreen indicating R24 

![image](https://github.com/Xinyuan-LilyGO/T-Display-S3-Long/assets/18565387/17a03bb7-75d4-4919-a56b-9d97754b8b39)

**Comment by lewisxhe** [comment]
Do you want to set the charging current to 3A? I do not recommend adjusting the charging current. It is reasonable to set the maximum to 500mA according to the circuit conditions.

**Comment by Grex44** [comment]
Yes or anything more than 500mA. I would like to increase the charging speed time of the battery that is currently being used. Maybe increase it to 1A or 2A at most.

**Comment by lewisxhe** [comment]
Please note that replacing this resistor carries the risk of damage, which you should be aware of
The picture below shows the location of the resistor
![image](https://github.com/Xinyuan-LilyGO/T-Display-S3-Long/assets/22990954/5918f015-ddc0-4f3d-a1fd-e1ac7cd61ebc)

**Comment by Grex44** [comment]
Thanks for the assistance. I will follow up again if there is any results upon the swap of resistor.
